### PR TITLE
Update BoxSnapshots.md

### PR DESCRIPTION
**Pull request by wilders-soccerfan**
update Snapshots.md and add image files to 'media' folder

### CODE REVIEW

**Comment by offhub** [comment]
Looks good. 👍

**Review by offhub — APPROVED** [review]

File: PlusContent/BoxSnapshots.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+![](../Media/Box_AutoDelete.png)
+
+**Installing Software to a Box and Creating a Snapshot:**
+- Select a box, disable AutoDelete, install the software to this box, set it up just the way you like.
+- Then, close the box, create a snapshot and enable box AutoDelete.
+- Now, this box will revert to the snapshot you created whenever it is closed. 
+
+**Updating Software Installed to a Box:**
+- Create a pre-update snapshot (for a baseline you can revert to, if need be-).
```
Comment (isaak654) [inline PlusContent/BoxSnapshots.md]:
```suggestion
- Create a pre-update snapshot (for a baseline you can revert to, if need be). 
```

I'm not sure about this hyphen.

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: PlusContent/BoxSnapshots.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+You have the ability to create a snapshot, remove a snapshot, revert to a snapshot or (starting with **Sandboxie Plus v1.0.9**) revert to an empty box while retaining all snapshots.
+
+**Caveat:** It is wise to use the snapshot features only for boxes whose location is on a real disk (and not on a ramdisk).
+
+**Additional Details:** 
+- Each snapshot is created its own folder, labeled snapshot-n,  where the number n is the snapshot id. You can change this label.
+- All snapshot folders for a given box are inside the box folder. 
+- The snapshot layout and information on the current (default) snapshot 
+is saved in the file **snapshot.ini** in the box folder.
```
Comment (isaak654) [inline PlusContent/BoxSnapshots.md]:
```suggestion
- The snapshot layout and information on the current (default) snapshot are saved in the file **snapshot.ini** in the box folder.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: PlusContent/BoxSnapshots.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Box Snapshots (for Sandboxie Plus)
+
+A **snapshot** saves the current state of a sandbox.
+You can create multiple snapshots of a box at different times and make one of the snapshots the default. 
+To get started, open the Sandman GUI, right-click on the desired sandbox and 
+click 'Snapshots Manager' from the drop-down list. See image below.
```
Comment (isaak654) [inline PlusContent/BoxSnapshots.md]:
```suggestion
To get started, open the Sandman GUI, right-click on the desired sandbox and click 'Snapshots Manager' from the drop-down list. See image below.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: PlusContent/BoxSnapshots.md (hunk 3 above)
Comment (wilders-soccerfan) [reply PlusContent/BoxSnapshots.md]: Agreed!

File: PlusContent/BoxSnapshots.md (hunk 1 above)
Comment (wilders-soccerfan) [reply PlusContent/BoxSnapshots.md]: Should not be there!

File: PlusContent/BoxSnapshots.md (hunk 2 above)
Comment (wilders-soccerfan) [reply PlusContent/BoxSnapshots.md]: Looks good. My grammar is weak.

File: PlusContent/BoxSnapshots.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Box Snapshots (for Sandboxie Plus)
+
+A **snapshot** saves the current state of a sandbox.
+You can create multiple snapshots of a box at different times and make one of the snapshots the default. 
+To get started, open the Sandman GUI, right-click on the desired sandbox and click 'Snapshots Manager' from the drop-down list. See image below.
```
Comment (isaak654) [inline PlusContent/BoxSnapshots.md]:
```suggestion
A **snapshot** saves the current state of a sandbox. You can create multiple snapshots of a box at different times and make one of the snapshots the default. To get started, open the Sandman GUI, right-click on the desired sandbox and click 'Snapshots Manager' from the drop-down list. See image below.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — APPROVED** [review]

**Review by isaak654 — CHANGES_REQUESTED** [review]
[Box_Snapshot2.png](https://raw.githubusercontent.com/sandboxie-plus/sandboxie-docs/db088731d1fe5ba0ca5d429c720497b212afc716/Media/Box_Snapshot2.png) is lower quality than the others two, could you produce a better one? 
I find the text not particularly crisp.

**Comment by wilders-soccerfan** [comment]
Hope the attached Box_Snapshot.png is satisfactory (the old one was from
David's post on Wilders).
Thank you for your painstaking attention to detail. Much appreciated. It
seems to have become a lost art in our hurried world.

**Comment by isaak654** [comment]
> Hope the attached Box_Snapshot.png is satisfactory

Unfortunately GitHub doesn't seem to attach images via email, it happened to other users as well.

**Review by wilders-soccerfan — COMMENTED** [review]
Box_Snapshot2

Hope this works.

**Comment by isaak654** [comment]
Please attach a new one with only the Snapshots dialog, to be taken specifically from the LockBox sandbox.

**Review by wilders-soccerfan — COMMENTED** [review]
Box_Snapshot2
Here it is.

**Review by isaak654 — APPROVED** [review]

**Pull request merged**